src/utility.ts

```diff
@@ -13,3 +16,9 @@ export const existsSync = (filename: string): boolean => {
     }
   }
 };
+
+export const generateTimestamp = (): string => {
+  const ts = (new Date()).toJSON();
+  globalThis.timeStamp = ts;
```

Comment:
can we remove this line? i dont see the need for globals, and i avoid them like a plague as they're generally bad practice in the javascript world